packages/vscode/languages/astro.code-snippets

```diff
@@ -0,0 +1,40 @@
+{
+	"page_html": {
+		"prefix": "page_html",
+		"isFileTemplate": true,
+		"scope": "astro",
+		"body": [
+			"---",
+			"",
```

Comment:
Though on placing the first cursor here? I'm thinking the first cursor should be here and then the other ones should match Emmett's `doc` snippet